docs/content/en/open_source/upgrading/2.39.md

```diff
@@ -2,7 +2,118 @@
 title: 'Upgrading to DefectDojo Version 2.39.x'
 toc_hide: true
 weight: -20240903
-description: No special instructions.
+description: Major upgrade of Postgres 16 to 17
 exclude_search: true
 ---
-There are no special instructions for upgrading to 2.39.x. Check the [Release Notes](https://github.com/DefectDojo/django-DefectDojo/releases/tag/2.39.0) for the contents of the release.
+
+# PostgreSQL Major Version Upgrade in Docker Compose
+
+This release incorporates a major upgrade of Postgres. When using the default docker compose setup you'll need to upgrade the Postgres data folder before you can use Defect Dojo 2.39.0.
+
+There are lots of online guides to be found such as https://hub.docker.com/r/tianon/postgres-upgrade or https://github.com/pgautoupgrade/docker-pgautoupgrade.
+
+There's also the [official documentation on `pg_upgrade`](https://github.com/pgautoupgrade/docker-pgautoupgrade), but this doesn't work out of the box when using Docker containers.
```

Comment:
Copy/paste error on that link - should be:

https://www.postgresql.org/docs/current/pgupgrade.html